solutions/CF Garland.mdx

```diff
@@ -2,13 +2,137 @@
 id: cf-garland
 source: CF
 title: An impassioned circulation of affection
-author: Neo Wang
+author: Neo Wang, Qi Wang, Jesse Choe
 ---
 
-[Official Editorial](https://codeforces.com/blog/entry/52449)
+[Official DP Editorial](https://codeforces.com/blog/entry/52449)
```

Comment:
how is this DP?

Reply:
wait thats mb, i thought nwatx put a dp editorial.